ChangeLog.md

```diff
@@ -33,6 +33,8 @@ See docs/process.md for more on how version tagging works.
   are now exposed to native code and can be used to keep the runtime alive
   without immediately unwinding the event loop (as
   `emscripten_exit_with_live_runtime()` does). (#17160)
+- Added a shim for `BigInt64Array` so `-sWASM_BIGINT` can be used in Safari
```

Comment:
This needs to move - 3.2.0 was renamed to 3.1.14 and merged meanwhile, so this will go into 3.1.15.